forge/src/main/java/org/openjdk/skara/forge/gitlab/GitLabForgeFactory.java

```diff
@@ -50,12 +50,12 @@ public Forge create(URI uri, Credential credential, JSONObject configuration) {
         if (configuration != null && configuration.contains("name")) {
             name = configuration.get("name").asString();
         }
-        Set<String> groups = new HashSet<>();
+        List<String> groups = new ArrayList<>();
```

Comment:
I would suggest using `List.of()` since we aren't updating this list.

Reply:
Sure. It's better.